.github/convert-files/conversion-scripts/asciidoc-to-epub.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+function convert_asciidoc_to_epub {
+  echo "Converting AsciiDoc files from ${ADOC_SOURCE_DIR} to EPUB..."
+
+  ADOC_MAIN_FILE="${ADOC_SOURCE_DIR}/${INPUT_BOOK_MAIN_FILE}.adoc"
+  EPUB_TARGET_FILE="${EPUB_TARGET_DIR}/${INPUT_BOOK_MAIN_FILE}.epub"
+
+  # Use the png files rather than svgs
+  for adoc in $(find . -name '*.adoc'); do
+    sed -i'.svg.bak' -e 's/\.svg/\.png/g' $adoc
+  done
+
+  echo "Converting ${ADOC_MAIN_FILE} to ${EPUB_TARGET_FILE}..."
+
+  asciidoctor-epub3 \
+      -a ebook-validate \
+      -a outlinelevels=4 \
+      -a series-name="Free5e" \
+      "${ADOC_MAIN_FILE}" \
+      -o "${EPUB_TARGET_FILE}"
+
+  echo "Checking the validity of ${EPUB_TARGET_FILE}..."
+  java -jar ${EPUBCHECK_PATH:-/free5e/epubcheck}/epubcheck.jar --version
+  java -jar ${EPUBCHECK_PATH:-/free5e/epubcheck}/epubcheck.jar "${EPUB_TARGET_FILE}"
```

Comment:
Notable change compared to the prior version: We now validate that the epub file is valid. If not, this job will fail.